README.md

```diff
@@ -25,6 +25,7 @@ A list of awesome compiler projects and papers for tensor computation and deep l
 - [Contribute](#contribute)
 
 ## Open Source Projects
```

Comment:
I propose to split this into two lists.

1. > 1000 github stars, sorted by github starts
2. < 1000 github stars, sorted by time

What do you think @yzh119

Reply:
I prefer only keeping list 1 projects and add notes to papers indicating whether their source code is available,